tests/modes.spec.ts

```diff
@@ -1,25 +1,132 @@
 import { readFile } from "fs/promises";
 import { test, expect, type Page } from "@playwright/test";
-import { loadInitialPage } from "./shared.ts";
+import {
+  loadInitialPage,
+  clickMap,
+  clearExistingInterventions,
+} from "./shared.ts";
```

Comment:
Fun new config problem: `npm run check` says `Error: An import path can only end with a '.ts' extension when 'allowImportingTsExtensions' is enabled.`

I tried removing the `.ts` to be consistent with the rest of our code, but then `npm run test` fails. Apparently a different tsconfig is being applied for tests?

Reply:
I'm not sure, I do remember   having some similar problem and just doing what got the tests to run and forgetting about it which might not be the best policy. I don't mind permitting importing with extensions but would be ideal to understand why this needs it and see if we can avoid